app/livechat/server/lib/Analytics.js

```diff
@@ -122,10 +155,14 @@ export const Analytics = {
 		Total_messages(date, departmentId) {
 			let total = 0;
 
-			LivechatRooms.getAnalyticsMetricsBetweenDate('l', date, { departmentId }).forEach(({ msgs }) => {
+			// we don't want to count visitor messages
+			const extraFilter = { $lte: ['$token', null] };
```

Comment:
Hi @KevLehman Just wanted to confirm about the logic we're adding here. Based on what I see in our docs, the **Total messages** count includes messages from visitor as well
![image](https://user-images.githubusercontent.com/34130764/129534787-abd7fe5c-e234-4e6f-99af-d7ad49d4de39.png)

Reply:
That's a great observation, let's get confirmation from @renatobecker about the expected behavior here, and if we should update code/docs. 

(IMO, It makes sense to show only user's messages on total messages, since that's an agent metric)

Reply:
Let's render only agent metrics and wait for the refactor of Analytics dashboards to improve it.